tasks/copy_files_to_archive/test/test_copy_files_to_archive.py

```diff
@@ -198,6 +200,10 @@ def test_handler_one_file_fail_3x(self):
         """
         Test copy lambda with one failed copy after 3 retries.
         """
+        del os.environ['COPY_RETRY_SLEEP_SECS']
```

Comment:
Nitpicky, but I don't think you need to `del` this value since you're just setting it in the next line. That `os.environ['COPY_RETRY_SLEEP_SECS'] = '3'` just overwrites the previous value.

`os.environ` is a [mapping](https://docs.python.org/3.3/glossary.html#term-mapping) type, which is apparently similar to a dictionary.

Reply:
Agreed. Tested, and it didn't make a difference. Removed here and elsewhere.